templates/zerver/integrations/git.md

```diff
@@ -1,31 +1,41 @@
+# Zulip Git integration
+
 Get Zulip notifications for your Git repositories!
 
+{start_tabs}
+
 1. {!create-an-incoming-webhook.md!}
 
 1. {!download-python-bindings.md!}
 
-1. {!create-channel.md!}
-
 1. {!change-zulip-config-file.md!}
 
-    You may also need to change the value of `STREAM_NAME`.
+    To customize further, refer to the
+    [configuration options](#configuration-options).
 
-    You can specify the branches that will be used for notifications by modifying
-    the `commit_notice_destination` function. By default,
-    pushes to the `main`, `master`, and `test-post-receive` branches will result in a
-    notification.
+1. Symlink the `post-receive` hook of your Git repository by running:
 
-1. Symlink `/usr/local/share/zulip/integrations/git/zulip_git_config.py`
-   to the `.git/hooks` directory of your Git repository.
-
-1. Symlink `/usr/local/share/zulip/integrations/git/post-receive`
-   to the `.git/hooks` directory of your Git repository.
+    `ln -s /usr/local/share/zulip/integrations/git/post-receive .git/hooks/post-receive`
 
 !!! tip ""
 
-    You can test the plugin without changing your `main` branch by
-    pushing to the `test-post-receive` branch.
+    Use the `test-post-receive` branch to test the plugin without modifying
+    your main branch.
+
+{end_tabs}
+
+### Configuration options
```

Comment:
This entire section has only been added in the last update.

Point 1: Inspired by the instruction removed from below the configuration code block, elaborate on it with the default value and a code snippet.

Point 2: Moved as-is from a tip above.

Point 3: Newly added option that I noticed in the config file.